src/playground/Widgets/List/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ProgressBar } from "@/components/Charts/ProgressChart"
+
+interface BaseListItem {
+  name: string
+}
+
+interface DefaultListItem extends BaseListItem {
+  value: number | string
+}
+
+interface ProgressListItem extends BaseListItem {
+  value: number
+}
+
+type ListItem = DefaultListItem | ProgressListItem
+
+interface ListProps {
+  items: ListItem[]
+  title?: string
+  variant?: "default" | "progress"
+}
+
+const ListItem = ({
+  item,
+  variant = "default",
+}: {
+  item: ListItem
+  variant?: "default" | "progress"
+}) => (
+  <div className="mb-2 flex w-full items-center last:mb-0">
+    <span className="flex-grow truncate text-muted-foreground">
+      {item.name}
+    </span>
+    {variant === "default" ? (
+      <span className="ml-4 flex-shrink-0 text-right text-sm font-medium">
```

Comment:
Do we need this `ml-4` here?

Reply:
With `flex-grow` in the span on the left maybe we won't need `text-right` either

Reply:
Right! Made a few changes here.